### ISSUE TITLE
After removing sandbox, librubberband won't build

### ISSUE THREAD

**Issue by nschlia** [rdp/ffmpeg-windows-build-helpers#544]
I've removed the sandbox to rebuild everything, but librubberband fails:

The log says:

```
 doing git checkout origin/master
 error: Pfadspezifikation 'origin/master' stimmt mit keinen git-bekannten Dateien überein.
```
The error message is that "origin/master" does not exist, actually this is a fact:
```
 $ git branch -r
  origin/HEAD -> origin/default
  origin/bqlibs
  origin/bqresample
  origin/cmake
  origin/default
  origin/freqmap
  origin/latencytest
  origin/meson
  origin/pitch-review
  origin/timing-review
```
Seems what usually is origin/master has been renamed to origin/default.  

Full part of that log:

```
08:34:16 Downloading (via git clone) rubberband_git from https://github.com/breakfastquay/rubberband.git
08:34:16 Klone nach 'rubberband_git.tmp' ...
08:34:16 done git cloning to rubberband_git
08:34:18 doing git checkout origin/master
08:34:18 error: Pfadspezifikation 'origin/master' stimmt mit keinen git-bekannten Dateien überein.
08:34:18 HEAD ist jetzt bei ae64f14 My mistake, I believe it's the target version
08:34:18 error: Pfadspezifikation 'origin/master' stimmt mit keinen git-bekannten Dateien überein.
08:34:18 fatal: mehrdeutiges Argument 'origin/master': unbekannter Commit oder Pfad existiert nicht
08:34:18 im Arbeitsverzeichnis
08:34:18 Benutzen Sie '--', um Pfade und Commits zu trennen, ähnlich wie:
08:34:18 'git <Befehl> [<Commit>...] -- [<Datei>...]'
08:34:18 Build step 'Shell ausführen' marked build as failure
08:34:18 Sending e-mails to: nschlia@oblivion-software.de
08:34:19 Finished: FAILURE
```

**Comment by nschlia** [comment]
Probably related to issue #545

**Comment by nschlia** [comment]
Fixed with commit 24c074f8f7321fa3a2fb71b377f956a71242ba52. Build ran successful. Thanks!
